components/exercises/chapter-5-oscar/Everything/some.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { some } from './some';
+
+describe('some', () => {
+  it('should return true if each satisfies the condition', () => {
+    const arr = [2, 4, 6, 8];
+    const callBack = (num: number) => num < 10;
+    expect(some(arr, callBack)).toBe(true);
+  });
+
+  it('should return false if at least one element satisfies the condition', () => {
+    const arr = [2, 4, 7, 8, 11];
+    const callBack = (num: number) => num < 10;
+    expect(some(arr, callBack)).toBe(false);
```

Comment:
there seems to be a bug here, since 2, 4,7 and 8 are < 10, this call should return true

Reply:
it hits false because 11 is bigger than 10, therefore it accomplish the condition